sources/loader/main.c

```diff
@@ -122,14 +124,16 @@ void loader_menu(void)
 {
     EFIInputKey key = efi_tty_get_key();
 
+    Buffer buffer;
+
+    auto entry = config_get_entry(str_cast("Brutal"), str_cast("/config.json"), &buffer);
+
     while (key.scan_code != SCAN_ESC)
     {
         if (key.unicode_char == CHAR_CARRIAGE_RETURN)
         {
             efi_tty_clear();
-            loader_boot(&(LoaderEntry){
-                .kernel = str_cast("/kernel.elf"),
-            });
+            loader_boot(&entry, &buffer);
```

Comment:
Why are you passing the buffer to loader_boot

Reply:
to deinit it after